packages/smart-wallet/src/smartWallet.js

```diff
@@ -255,6 +257,8 @@ export const makeSmartWallet = async (
      * This version does not: 1) for expedience, 2: to avoid resource exhaustion vulnerability.
      */
     receive: async (paymentE, paymentBrand) => {
+      fit(harden(paymentE), M.eref(PaymentShape));
```

Comment:
What is the interface over which these are being received?  should that just be a FarClass? Not blocking, but might be simpler.

Reply:
it will be. that's the next PR.